lib/Doctrine/ORM/Tools/Console/Command/SchemaTool/UpdateCommand.php

```diff
@@ -124,35 +132,48 @@ protected function executeSchemaCommand(InputInterface $input, OutputInterface $
         $force   = true === $input->getOption('force');
 
         if ($dumpSql) {
-            $output->writeln(implode(';' . PHP_EOL, $sqls) . ';');
+            $this->io->text('The following SQL statements will be executed:');
+            $this->io->newLine();
+
+            foreach ($sqls as $sql) {
+                $this->io->text(sprintf('    %s;', $sql));
+            }
         }
 
         if ($force) {
             if ($dumpSql) {
-                $output->writeln('');
+                $this->io->newLine();
             }
-            $output->writeln('Updating database schema...');
+            $this->io->text('Updating database schema...');
+            $this->io->newLine();
+
             $schemaTool->updateSchema($metadatas, $saveMode);
 
             $pluralization = (1 === count($sqls)) ? 'query was' : 'queries were';
 
-            $output->writeln(sprintf('Database schema updated successfully! "<info>%s</info>" %s executed', count($sqls), $pluralization));
+            $this->io->text(sprintf('    <info>%s</info> %s executed', count($sqls), $pluralization));
+            $this->io->success('Database schema updated successfully!');
         }
 
         if ($dumpSql || $force) {
             return 0;
         }
 
-        $output->writeln('<comment>ATTENTION</comment>: This operation should not be executed in a production environment.');
-        $output->writeln('           Use the incremental update to detect changes during development and use');
-        $output->writeln('           the SQL DDL provided to manually update your database in production.');
-        $output->writeln('');
+        $this->io->caution(<<<EOT
```

Comment:
In case you prefer it, most methods accept both strings and arrays of strings, so this code could be:

```php
$this->io->caution([
    'This operation should not be executed in a production environment!',
    '',
    'Use the incremental update to detect changes during development and use',
    'the SQL DDL provided to manually update your database in production.',
]);
```

Reply:
I saw this afterwards 👍 

I can rework this if the guys like the PR and prefer the array syntax.

Thank you for your feedback!

Reply:
Array syntax looks better in my opinion

Reply:
done